packages/core/src/browser/frontend-application.ts

```diff
@@ -170,6 +170,34 @@ export class FrontendApplication {
         return 0;
     }
 
+    /**
+     * Initialize the shell layout either using the layout restorer service or, if no layout has
+     * been stored, by creating the default layout.
+     */
+    protected async initializeLayout(): Promise<void> {
+        try {
+            // Try to restore the shell layout from the storage service
+            await this.layoutRestorer.restoreLayout(this);
```

Comment:
Before this change, we have `await app.shell.pendingUpdates;` in the `try-catch` to make sure errors that happen during the restore are caught. Could you please explain what is the equivalence in your change?